### ISSUE TITLE
Muokkaa 035a-kentän aktivoitumista valutuksessa.

### ISSUE THREAD

**Issue by johannaraisa** [KohaSuomi/koha-plugin-broadcast-biblios#13]
Jos 035a-kentässä on BTJ:n arvo ja Tätissä ei ole vastaavaa arvoa, tietueeseen ei valu uuttaa tietoa.

**Comment by johannaraisa** [comment]
Siivosin koodin entiselleen tuon BTJ:n 035a-kentän aktivoitumisen osalta. Tein tätä varten skriptin millä pystyy asettamaan uudet arvot noille FI-BTJ aktivoinneille.

**Comment by AnttHeik** [comment]
Hei, vaikuttiko tämänaamuinen päivitys siihen, ettei kaikkiin hankintatietueisiin valunut tietueita? Katselin OUTIssa paria tietuetta, joilla oli aktivointiarvona 035-kentän BTJ-tunnus, mutta aktivointiarvo vaihtui sitten kun toin tietueen TäTistä tietuesiirtäjällä: 2372863 ja ehkä myös tämä: 2372655

**Comment by johannaraisa** [comment]
Joo, tuo on todennäköisin syy, kun ei enää käsitellä niitä FI-BTJ-arvoja. Laitan tuon päiväpaketin uudestaan liikkeelle.

**Comment by johannaraisa** [comment]
Sain korjattua nuo FI-BTJ aktivointiarvot tietokannoista, joten niiden pitäisi napata kiinni muilla arvoilla.

**Comment by AnttHeik** [comment]
Jeps, kiitokset. Ainakin OUTIssa tuli nyt uudestaan tämänpäiväiset.
